### PR TITLE
Feature/noshin

### PR DESCRIPTION
**Pull request by noshinbashar**
@trupti-patil23 @cathybi @timho1985 @LigangMa2022

installed react-router-dom
created react routers in App.jsx file
imported components in App.jsx file
created EditInventory.jsx file in EditInventory component (when I was importing components in app.jsx I found it missing, that's why I created it)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,7 +1,33 @@
 import "./App.scss";
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import InventoryPage from "./pages/InventoryPage/InventoryPage"
+import WarehousesPage from "./pages/WarehousesPage/WarehousesPage"
+import AddNewInventory from "./components/AddNewInventory/AddNewInventory"
```
Comment (timho1985) [inline src/App.jsx:5]: Do we need components in App.jsx page?

Comment (noshinbashar) [reply src/App.jsx:5]: @cathybi we need components in App.jsx page. right?

Comment (noshinbashar) [reply src/App.jsx:5]: Hey @timho1985 we need those in App.jsx

**Review by cathybi — COMMENTED** [review]
Looks good to me

**Review by timho1985 — APPROVED** [review]

**Pull request closed without merging**